interpreter/cling/lib/Interpreter/DeclCollector.cpp

```diff
@@ -57,6 +58,15 @@ namespace {
         return true;
     return false;
   }
+
+  /// \brief Asserts that the given transaction is not null, otherwise prints a
+  /// stack trace to stderr.
+  static void assertHasTransaction(const cling::Transaction* transaction) {
```

Comment:
Using just T as a variable name is a common notion for variable names of type cling::Transaction.